reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -70,7 +70,10 @@
 @click.option('--containerized', default='False', help='Containerized installation of OpenShift',
               show_default=True)
 @click.option('--s3-bucket-name', help='Bucket name for S3 for registry')
-@click.option('--s3-username', help='S3 user for registry access')
+@click.option('--github-client-id', help='GitHub OAuth ClientID')
+@click.option('--github-client-secret', help='GitHub OAuth Client Secret')
+@click.option('--organization', help='GitHub Organization(Comma Seperated Values)')
```

Comment:
I'd suggest using `--github-organization` to be more explicit.

Also, instead of using a comma separated value, you can configure the option with `multiple=True` to allow for the user to specify it multiple times: http://click.pocoo.org/5/options/#multiple-options